pkgs/development/tools/rgp/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib, stdenv, makeWrapper, fetchurl, autoPatchelfHook,
+  dbus, fontconfig, freetype, glib, libglvnd, libX11, libxcb, libXi, ncurses5,
```

Comment:
Minor nitpick: Its common practice in nixpkgs to add the `,` to the next line when splitting lines. Even better would be to put each argument in its own line, e.g.

```
{ lib
, stdenv
, makeWrapper
...
```